docker-compose/cassandra-3.11/docker-compose-dev-mode.yml

```diff
@@ -36,9 +35,9 @@ services:
       - "8082:8082"
     mem_limit: 2G
     environment:
-      - QUARKUS_HTTP_CORS=true
-      - QUARKUS_HTTP_CORS_ORIGINS=*
-      - QUARKUS_HTTP_CORS_METHODS=POST,GET,PUT,OPTIONS,DELETE
+      #- QUARKUS_HTTP_CORS=true
+      #- QUARKUS_HTTP_CORS_ORIGINS=*
+      #- QUARKUS_HTTP_CORS_METHODS=POST,GET,PUT,OPTIONS,DELETE
```

Comment:
Also tried disabling the cors filtering to see if that was the cause of the socket close, but does not seem to be the issue.